raven/src/main/java/com/getsentry/raven/event/Breadcrumb.java

```diff
@@ -12,17 +12,21 @@
 public class Breadcrumb implements Serializable {
 
     /**
+     * Default category.
+     */
+    private static final String DEFAULT_CATEGORY = "generic";
```

Comment:
It's what the server defaults to, but it just dawned on me that we should just leave these `null` by default and let the server do its thing. I'm out of it today. 😷